tests/settings.py

```diff
@@ -24,3 +24,5 @@
     TEST_RUNNER = 'discover_runner.DiscoverRunner'
 
 SECRET_KEY = 'spam-spam-spam-spam'
+
+MIDDLEWARE_CLASSES = ()
```

Comment:
Without this setting, the following system check warning is printed many times during the test run. Hiding this warning makes it easier to see the real errors and warnings (if there are any).

```
System check identified some issues:

WARNINGS:
?: (1_7.W001) MIDDLEWARE_CLASSES is not set.
	HINT: Django 1.7 changed the global defaults for the MIDDLEWARE_CLASSES.
    django.contrib.sessions.middleware.SessionMiddleware,
    django.contrib.auth.middleware.AuthenticationMiddleware, and
    django.contrib.messages.middleware.MessageMiddleware were removed from the  
    defaults. 
    If your project needs these middleware then you should configure this setting.
```